packages/core/src/html/includePanelProcessor.ts

```diff
@@ -230,6 +240,26 @@ export function processInclude(node: MbNode, context: Context, pageSources: Page
     const $ = cheerio.load(actualContent);
     const actualContentOrNull = $(hash).html();
     actualContent = actualContentOrNull || '';
+    if (actualContent !== '') {
+      const hashNode = $(hash);
+      const footnodeHrefs = hashNode.find('a[aria-describedby="footnote-label"]')
+        .map(function (this: any) {
+          $(this).text(`[${getNextFootnodeNumber()}]`);
+          return $(this).attr('href');
+        })
+        .get();
+      if (footnodeHrefs.length > 0) {
+        const tempFootnotes = $('<mb-temp-footnotes></mb-temp-footnotes>');
+        footnodeHrefs.forEach((href) => {
+          const listItem = $('<li></li>').attr('id', href.substring(1))
```

Comment:
Could you add a comment to explain why we need to do a substring here? It is just to get rid of the hash in front right?

Reply:
Thank you for the comment; has added the the latest commit